openmetadata-ui/src/main/resources/ui/src/components/tags/tags.tsx

```diff
@@ -45,6 +45,10 @@ const Tags: FunctionComponent<TagProps> = ({
     return tag.startsWith('#') ? tag.slice(1) : tag;
   };
 
+  const getDisplayTag = (tag: string) => {
+    return tag.length > 35 ? `${tag.slice(0, 35)}...` : tag;
+  };
+
```

Comment:
Can we move this function to util and name it `getTagDisplay`?
Also, can we use `viewCap` from constant?
```js
const TAG_VIEW_CAP = 35;
```